Sprint-1/exercises/random.js

```diff
@@ -1,8 +1,12 @@
 const minimum = 1;
 const maximum = 100;
 
+// Generate a random number between minimum and maximum
```

Comment:
Can you explain why the expression uses `(maximum - minimum + 1)` and not `(maximum - minimum)`?

If you were to write an expression that evaluates to a random integer between -5 and 3 (including both -5 and 3), how would you write such expression?

Reply:
The expression (maximum - minimum + 1) ensures the range includes both minimum and maximum. Without +1, the upper bound would be excluded.

To generate a random integer between -5 and 3 (inclusive), I can do this:
const randomNum = Math.floor(Math.random() * (3 - (-5) + 1)) + (-5);